docs/source/devguide/build_docs.rst

```diff
@@ -12,4 +12,4 @@
 
 **********************
 Building documentation
-**********************
\ No newline at end of file
+**********************
```

Comment:
What happened here? The lines look identical. Changed line ending character?

Reply:
I thought that github marked removed whitespace with slightly darker pink, but that doesn't seem to be the case anymore. This removed a trailing space.